### PR TITLE
add Fetch action

### PR DESCRIPTION
**Pull request by goneri**
`vl fetch $foo` allow the user to quickly import an existing image.

### CODE REVIEW

File: virt_lightning/shell.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 import argparse
 import asyncio
 import logging
+import http.client
```
Comment (zloidemon) [inline virt_lightning/shell.py]: Did you see urllib?

File: virt_lightning/shell.py [hunk 2]
```diff
@@ -335,6 +337,48 @@ def storage_dir(configuration, **kwargs):
...
+        print(  # noqa: T001
+            "File already exists: {target_file}".format(target_file=target_file)
+        )
+        sys.exit(0)
+    with temp_file.open("wb") as fd:
+        while fd.tell() < lenght:
+            chunk = r.read(chunk_size)
+            fd.write(chunk)
+            percent = (fd.tell() * 100) / lenght
+            line = "[{percent:06.2f}%]  {done:6}MB/{full}MB\r".format(
```
Comment (zloidemon) [inline virt_lightning/shell.py]: This code might change to use quantiphy library.

File: virt_lightning/shell.py (hunk 1 above)
Comment (goneri) [reply virt_lightning/shell.py]: I don't want any extra dependence. I can use urllib.request instead. What do you think?

File: virt_lightning/shell.py (hunk 2 above)
Comment (goneri) [reply virt_lightning/shell.py]: I would prefer to avoid any third-party dep for now.

**Pull request merged**